runtime/src/append_vec.rs

```diff
@@ -301,7 +301,7 @@ impl AppendVec {
         self.file_size
     }
 
-    pub fn file_name(slot: Slot, id: usize) -> String {
+    pub fn file_name(slot: Slot, id: impl std::fmt::Display) -> String {
```

Comment:
Why not use `AppendVecId` as the type here too? With the `impl Display`, that could allow a lot of types.

Reply:
My thinking was this is not including anything else from the crate in it. Seems like a leaf, simple module. And we just need an id to format. So, instead of creating a circular dependency, I chose to keep the dependency graph is it is.

Reply:
Oh! I didn't know that `AppendVecId` wasn't in `append_vec.rs`. I feel like that's counterintuitive (at least for me). Would it be a big pain to move `AppendVecId` into this `append_vec.rs`?

Reply:
Ok, nevermind, I see, it's not part of the AppendVec. Sorry about that, I was confused.